src/sentry/features/__init__.py

```diff
@@ -178,6 +178,7 @@
 default_manager.add("organizations:set-grouping-config", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:slack-escape-messages", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:slack-overage-notifications", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:sdk-crash-reporting", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
We should not use externally managed feature flags in ingestion paths (rather specify a dedicated feature handler in getsentry):

```suggestion
default_manager.add("organizations:sdk-crash-reporting", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

During local development, you can set `SENTRY_FEATURES` manually either way.